lib/services/platform-service.ts

```diff
@@ -368,15 +368,56 @@ export class PlatformService implements IPlatformService {
 				this.$errors.failWithoutHelp("Unable to find built application. Try 'tns build %s'.", platform);
 			}
 
-			this.$fs.ensureDirectoryExists(path.dirname(targetPath)).wait();
+			function ext(file: string): string {
+				return path.extname(file).toLowerCase();
+			}
+			function targetExt(file: string): string {
+				let e = ext(file);
+				return (e === ".ipa" || e === ".app") ? e : ".";
```

Comment:
ok, I do not have better ideas as well, so let's keep it simple (use `.`) :smiley_cat: